src/data/pnat/basic.lean

```diff
@@ -23,6 +23,12 @@ instance : has_repr ℕ+ := ⟨λ n, repr n.1⟩
 /-- Predecessor of a `ℕ+`, as a `ℕ`. -/
 def pnat.nat_pred (i : ℕ+) : ℕ := i - 1
 
+@[simp] lemma pnat.one_add_nat_pred (n : ℕ+) : 1 + n.nat_pred = n :=
+by rw [pnat.nat_pred, add_tsub_cancel_iff_le.mpr $ show 1 ≤ (n : ℕ), from n.2]
```

Comment:
As part of a brief unsuccessful attempt to golf this, I introduced:
```lean
@[simp] lemma pnat.nat_pred_eq_pred {n : ℕ} (h : 0 < n) :
  pnat.nat_pred (⟨n, h⟩ : ℕ+) = n.pred :=
rfl
```
I think maybe worth adding while you're here.